native/Makefile

```diff
@@ -410,16 +433,18 @@ endif
 PCFLAGS+=$(ISYSROOT) -x objective-c
 # JAWT no longer supported on OSX
 CDEFINES+=-DTARGET_RT_MAC_CFM=0 -DFFI_MMAP_EXEC_WRIT -DNO_JAWT
-LDFLAGS=$(ARCHFLAGS) -dynamiclib -o $@ -framework JavaVM \
+LDFLAGS=$(ARCHFLAGS) -dynamiclib -o $@ \
   -compatibility_version $(shell echo ${JNA_JNI_VERSION}|sed 's/^\([0-9][0-9]*\).*/\1/g') \
   -current_version $(JNA_JNI_VERSION) \
   -mmacosx-version-min=10.3 \
   -framework Foundation \
   $(NO_COMPACT_UNWIND) \
   -install_name ${@F} \
   $(SYSLIBROOT)
+ifeq ($(shell test $(XCODE_VERSION) -lt 12; echo $$?),0)
 # JAWT linkage handled by -framework JavaVM
-LIBS=
+LIBS+=-framework JavaVM
+endif
```

Comment:
Is linking against JavaVM not required anymore?

Reply:
Apple does not ship a `JavaVM` framework anymore.

Reply:
So do we need the JavaVM framework? Or could we drop it from the build entirely?

Reply:
It seems to me this was only required for JAWT. Looks to me like it could be dropped for non-arm64 as well.